src/_pytest/timing.py

```diff
@@ -0,0 +1,11 @@
+"""
+Indirection for time functions.
+
+We intentionally grab some "time" functions internally to avoid tests mocking "time" to affect
+pytest runtime information (issue #185).
+
+Fixture "mock_timinig" also interacts with this module for pytest's own tests.
+"""
+from time import perf_counter  # noqa: F401
```

Comment:
If you use `__all__`, you can avoid the noqa's.

Reply:
Ahh indeed, thanks!